pyQuARC/schemas/rule_mapping.json

```
@@ -4592,4 +4614,4 @@
         "severity": "info",
         "check_id": "date_compare"
     }
-}
+}
```

```suggestion
}

```

I think it's good practice to add a new line to the end of the file